kimchi/src/folding/error.rs

```diff
@@ -0,0 +1,331 @@
+use crate::folding::{
```

Comment:
Can you add a doc-comment to the top of this module? It wasn't immediately obvious that this is about the error terms for folding. Perhaps renaming to `error_term.rs` would help too.